wcfsetup/install/files/lib/data/user/User.class.php

```diff
@@ -34,7 +34,8 @@
  * @property-read	integer		$banned				is `1` if the user is banned, otherwise `0`
  * @property-read	string		$banReason			reason why the user is banned
  * @property-read	integer		$banExpires			timestamp at which the banned user is automatically unbanned
- * @property-read	integer		$activationCode			code sent to the user's email address used for account activation
+ * @property-read	integer		$activationCode			flag which determines, whether the user is activated (for legacy reasons an random integer, if the user is *not* activated)
+ * @property-read	string		$emailConfirmed			code sent to the user's email address used for account activation
```

Comment:
```suggestion
 * @property-read	string		$emailConfirmed			Code sent to the user's email address used for account activation. The empty string if the email is confirmed.
```

Reply:
`The empty string if the email is confirmed.` should be replaced with `or empty if the email is confirmed.` to keep the wording consistent.

Reply:
We are not using full sentence in the parameter description there. I would add `or empty if the email is confirmed` at the end of the description, if you are fine with this @TimWolla

Reply:
I'm fine with that.